### PR TITLE
Add sip specific custom error pages

### PR DESCRIPTION
**Pull request by spr06**
Items:
====

1) Add sip specific custom error pages
2) Add a switch to selectively display contact information.
    The switch name is: DISPLAY_CONTACT_DETAILS, when set to TRUE will display custom contact information

### CODE REVIEW

File: html/partials/500.shtml [hunk 1]
```diff
@@ -1,2 +1,10 @@
-<h2>500 Internal server error</h2>
-<p>There's a problem. Please try again in a few moments.</p>
+<p><strong><h1></h1>Something's gone wrong with our servers</h1></strong></p>
```
Comment (daniel-ac-martin) [inline html/partials/500.shtml]: This strikes me as bad HTML relative to what it is replacing.

File: html/partials/500.shtml [hunk 2]
```diff
@@ -1,2 +1,10 @@
...
+<p>We're fixing this as quickly as possible.</p>
+<p>Please try again.</p>
+<p>If you're still having difficulty:</p>
```
Comment (daniel-ac-martin) [inline html/partials/500.shtml]: The content was only recently agreed with Fola and the rest of the content designers. What is the need to change it?

In particular, it seems odd that we would state that we are working to fix the problem (are we?) but also asking the user to try again.

File: html/partials/500.shtml [hunk 3]
```diff
@@ -1,2 +1,10 @@
...
+<!--#if expr="$display_contact_details = TRUE" -->
+<ul>
+	<li>phone the IT service desk on 0845 00 00 050 or;</li>
+	<li>email ITServiceDesk@homeoffice.gsi.gov.uk</li>
```
Comment (daniel-ac-martin) [inline html/partials/500.shtml]: These details will be specific to your project. It is not acceptable to these to be hard-coded into this repo as it is used my multiple projects.

Instead we should allow users to set their own contact details via environment variables.

File: html/partials/500.shtml (hunk 1 above)
Comment (deed02392) [reply html/partials/500.shtml]: `<h1></h1>Something's...</h1>`

**Pull request closed without merging**